presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -55,7 +54,6 @@
 #include "velox/serializers/PrestoSerializer.h"
 
 #ifdef PRESTO_ENABLE_REMOTE_FUNCTIONS
-#include "presto_cpp/main/RemoteFunctionRegisterer.h"
```

Comment:
Why is this header being removed?

Reply:
the clion highlights as unused. So I removed it. Added back.